protocol/synthetix/subgraph/subgraph.arbitrum-sepolia.yaml

```
@@ -0,0 +1,100 @@
+specVersion: 0.0.4
+schema:
+  file: ./arbitrum-sepolia/schema.graphql
+dataSources:
+  - kind: ethereum
+    network: arbitrum-one
+    name: CoreProxy
+    source:
+      abi: CoreProxy
+      address: "0x4Ab09F627fa9706f33178129005300Fd4c00EDF1"
+      startBlock: 190322190
```

https://sepolia.arbiscan.io/address/0x76490713314fCEC173f44e99346F54c6e92a8E42

```suggestion
      address: "0x76490713314fCEC173f44e99346F54c6e92a8E42"
      startBlock: 41772504
```